assets/javascripts/discourse/components/ai-persona-editor.gjs

```diff
@@ -103,26 +123,42 @@ export default class PersonaEditor extends Component {
 
   @action
   async toggleEnabled() {
-    this.args.model.set("enabled", !this.args.model.enabled);
-    this.editingModel.set("enabled", this.args.model.enabled);
-    if (!this.args.model.isNew) {
-      try {
-        await this.args.model.update({ enabled: this.args.model.enabled });
-      } catch (e) {
-        popupAjaxError(e);
-      }
-    }
+    this.toggleField("enabled");
   }
 
   @action
   async togglePriority() {
-    this.args.model.set("priority", !this.args.model.priority);
-    this.editingModel.set("priority", this.args.model.priority);
+    this.toggleField("priority", true);
+  }
+
+  @action
+  async toggleMentionable() {
```

Comment:
what's the goal of async here?

Reply:
I guess cause toggleField is async and I wanted to signal that... I think maybe I should await here which at least will clarify this.

Reply:
If you don't await, I don't think it awaits, it will just return a promise that's all